src/c++/perf_analyzer/inference_profiler.h

```diff
@@ -594,6 +594,24 @@ class InferenceProfiler {
     }
   }
 
+  template <typename T>
+  void GetMetricFirstPerGPU(
+      const std::vector<std::reference_wrapper<const std::map<std::string, T>>>&
```

Comment:
Does it make sense to use a map where the first argument is GPU UUID and the second argument is a vector? This can apply to other GetMettric<AGG>PerGPU too?

Reply:
Discussed offline and decided that this parameter was designed ok enough as-is